libdnf/base/base.cpp

```diff
@@ -23,6 +23,8 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "libdnf/conf/const.hpp"
 #include "solv/pool.hpp"
 
+#include "utils/bgettext/bgettext-lib.h"
```

Comment:
This include looks redundant.

Reply:
`M_` is used on line 72: `std::throw_with_nested(RuntimeError(M_("Unable to load configuration file \"{}\""), path));` .
Yes, it is currently possible to compile without this include, as it will get here via "solv/pool.hpp". I'd rather keep the include here.

Reply:
Ah, sure, sorry, overlooked.